public/css/bootstrap.scss

```diff
@@ -0,0 +1,100 @@
+@import "../node_modules/bootstrap/scss/functions";
+
+// Variable overrides
+
+// Enable shadows and gradients.  These are disabled by default.
+$enable-shadows: true;
+
+// Use a smaller grid gutter width.  The default is 1.5rem.
+$grid-gutter-width: 1rem;
+
+// Fonts
+$font-size-base: 0.85rem;
+$headings-font-weight: 600;
+
+// Links
+$link-decoration: none;
+$link-hover-decoration: underline;
+
+// Make breadcrumb dividers and active items a bit darker.
+$breadcrumb-divider-color: #495057;
+$breadcrumb-active-color: #495057;
+
+@import "./theme-colors";
+
+// Include the remainder of bootstrap's scss configuration
+@import "../node_modules/bootstrap/scss/variables";
+@import "../node_modules/bootstrap/scss/maps";
+@import "../node_modules/bootstrap/scss/mixins";
+@import "../node_modules/bootstrap/scss/utilities";
+
+// Layout & components
+@import "../node_modules/bootstrap/scss/root";
+@import "../node_modules/bootstrap/scss/reboot";
+@import "../node_modules/bootstrap/scss/type";
+@import "../node_modules/bootstrap/scss/images";
+@import "../node_modules/bootstrap/scss/containers";
+@import "../node_modules/bootstrap/scss/grid";
+@import "../node_modules/bootstrap/scss/tables";
+@import "../node_modules/bootstrap/scss/forms";
+@import "../node_modules/bootstrap/scss/buttons";
+@import "../node_modules/bootstrap/scss/transitions";
+@import "../node_modules/bootstrap/scss/dropdown";
+@import "../node_modules/bootstrap/scss/button-group";
+@import "../node_modules/bootstrap/scss/nav";
+@import "../node_modules/bootstrap/scss/navbar";
+@import "../node_modules/bootstrap/scss/card";
+@import "../node_modules/bootstrap/scss/accordion";
+@import "../node_modules/bootstrap/scss/breadcrumb";
+@import "../node_modules/bootstrap/scss/pagination";
+@import "../node_modules/bootstrap/scss/badge";
+@import "../node_modules/bootstrap/scss/alert";
+@import "../node_modules/bootstrap/scss/placeholders";
+@import "../node_modules/bootstrap/scss/progress";
+@import "../node_modules/bootstrap/scss/list-group";
+@import "../node_modules/bootstrap/scss/close";
+@import "../node_modules/bootstrap/scss/toasts";
+@import "../node_modules/bootstrap/scss/modal";
+@import "../node_modules/bootstrap/scss/tooltip";
+@import "../node_modules/bootstrap/scss/popover";
+@import "../node_modules/bootstrap/scss/carousel";
+@import "../node_modules/bootstrap/scss/spinners";
+@import "../node_modules/bootstrap/scss/offcanvas";
+
+// Helpers
+@import "../node_modules/bootstrap/scss/helpers";
+
+// Utilities
+@import "../node_modules/bootstrap/scss/utilities/api";
+
+// WeBWorK specific colors
+:root {
+	--ww-logo-background-color: #{$ww-logo-background-color};
+	--ww-primary-foreground-color: #{color-contrast($primary)};
+	--ww-achievement-level-color: #{$ww-achievement-level-color};
+}
+
+// Overrides
+a:not(.btn):focus {
+	color: $link-hover-color;
+	outline-style: solid;
+	outline-color: $link-hover-color;
+	outline-width: 1px;
+}
+
+@import "theme-overrides";
```

Comment:
This should be removed since the `_theme-overrides.scss` file does not exist for the renderer.